aiinfra-cluster/modules/dashboard/main.tf

```diff
@@ -0,0 +1,216 @@
+  description  = "Fraction of resident warps on a multiprocessor relative to the maximum number supported, averaged over time and all multiprocessors."
+  metric_kind  = "GAUGE"
+  value_type   = "DOUBLE"
+  unit         = "1"
+  labels {
+    key         = local.attributes.model.key
+    description = local.attributes.model.description
+  }
+  labels {
+    key         = local.attributes.gpu_number.key
+    description = local.attributes.gpu_number.description
+  }
+  labels {
+    key         = local.attributes.uuid.key
+    description = local.attributes.uuid.description
+  }
+  launch_stage = "BETA"
+  metadata {
+    sample_period = "60s"
+    ingest_delay  = "30s"
+  }
+}
+
+resource "google_monitoring_metric_descriptor" "pipe_utilization" {
+  display_name = "dcgm.gpu.pipe_utilization"
+  type         = "workload.googleapis.com/dcgm.gpu.pipe_utilization"
+  description  = "Fraction of cycles the corresponding GPU pipe was active, averaged over time and all multiprocessors."
+  metric_kind  = "GAUGE"
+  value_type   = "DOUBLE"
+  unit         = "1"
+  labels {
+    key         = local.attributes.model.key
+    description = local.attributes.model.description
+  }
+  labels {
+    key         = local.attributes.gpu_number.key
+    description = local.attributes.gpu_number.description
+  }
+  labels {
+    key         = local.attributes.uuid.key
+    description = local.attributes.uuid.description
+  }
+  labels {
+    key         = local.attributes.pipe.key
+    description = local.attributes.pipe.description
+  }
+  launch_stage = "BETA"
+  metadata {
+    sample_period = "60s"
+    ingest_delay  = "30s"
+  }
+}
+
+resource "google_monitoring_metric_descriptor" "dram_utilization" {
+  display_name = "dcgm.gpu.dram_utilization"
+  type         = "workload.googleapis.com/dcgm.gpu.dram_utilization"
+  description  = "Fraction of cycles data was being sent or received from GPU memory."
+  metric_kind  = "GAUGE"
+  value_type   = "DOUBLE"
+  unit         = "1"
+  labels {
+    key         = local.attributes.model.key
+    description = local.attributes.model.description
+  }
+  labels {
+    key         = local.attributes.gpu_number.key
+    description = local.attributes.gpu_number.description
+  }
+  labels {
+    key         = local.attributes.uuid.key
+    description = local.attributes.uuid.description
+  }
+  launch_stage = "BETA"
+  metadata {
+    sample_period = "60s"
+    ingest_delay  = "30s"
+  }
+}
+
+resource "google_monitoring_metric_descriptor" "pcie_traffic_rate" {
+  display_name = "dcgm.gpu.pcie_traffic_rate"
+  type         = "workload.googleapis.com/dcgm.gpu.pcie_traffic_rate"
+  description  = "The average rate of bytes sent from the GPU over the PCIe bus over the sample period, including both protocol headers and data payloads."
+  metric_kind  = "GAUGE"
+  value_type   = "INT64"
+  unit         = "By/s"
+  labels {
+    key         = local.attributes.model.key
+    description = local.attributes.model.description
```

Comment:
it does not have description i believe when auto generated. 

it has to match exactly otherwise Ops agent wont be able to send data points. Thats why I was thinking we should just have the 2 metric descriptor in the hotfix, those are causing issues.